abyss/Dockerfile

```diff
@@ -0,0 +1,11 @@
+# DOCKERFILE for abyss
+FROM bcgsc/abyss:latest
```

Comment:
Although this is perfectly fine. It would be better that you bind this dockerfile to a release with a more specific tag name (e.g. `2.0.0-1`) instead of `latest`, which will always fetch the last one. However, if this is intentional don't mind this comment but be aware that if the `lastest` "parent" docker container changes this one will also be affected.

Reply:
There's a bug in the docker image for `bcgsc/abyss:2.1.1` that will be resolved in the next docker image, but the bug doesn't exist in `bcgsc/abyss:latest`. As soon as the docker image is fixed, I can change the tag to a more specific tag.